omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_0/UserResource2_0Test.java

```diff
@@ -195,4 +197,29 @@ public void testSearchingByUserAndRole() {
 		assertSearch("doesNotExist", Arrays.asList("System Developer"), 0);
 	}
 	
+	@Test
+	public void testChangeUsersOwnPassword() throws Exception {
```

Comment:
I don't think this test is valid, because it is run as the admin user.
See https://github.com/openmrs/openmrs-module-hr/blob/d7fbd21389d4998f2a8aa7885f81dcc10a507474/api/src/test/java/org/openmrs/module/hr/api/HrPostServiceTest.java#L64 for an example of how to do, i.e. using `@SkipBaseSetup` and `Context.authenticate` so that you can become a different user.